src/main/scala/io/getclump/Clump.scala

```diff
@@ -19,12 +19,37 @@ sealed trait Clump[+T] {
 
   def handle[B >: T](f: PartialFunction[Throwable, Option[B]]): Clump[B] = new ClumpHandle(this, f)
 
+  /**
+   * Alias for [[handle]]
+   */
+  def recover[B >: T](f: PartialFunction[Throwable, Option[B]]): Clump[B] = handle(f)
+
   def rescue[B >: T](f: PartialFunction[Throwable, Clump[B]]): Clump[B] = new ClumpRescue(this, f)
 
+  /**
+   * Alias for [[rescue]]
+   */
+  def recoverWith[B >: T](f: PartialFunction[Throwable, Clump[B]]): Clump[B] = rescue(f)
+
+  /**
+   * On any exception, fallback to a default value
+   */
+  def fallback[B >: T](default: => Option[B]): Clump[B] = handle(PartialFunction(_ => default))
+
+  /**
+   * On any exception, fallback to a default clump
+   */
+  def fallbackTo[B >: T](default: => Clump[B]): Clump[B] = rescue(PartialFunction(_ => default))
+
   def withFilter[B >: T](f: B => Boolean): Clump[B] = filter(f)
 
   def filter[B >: T](f: B => Boolean): Clump[B] = new ClumpFilter(this, f)
 
+  /**
+   * If this clump does not return a value then use the default instead
+   */
+  def default[B >: T](default: => B): Clump[B] = new ClumpOrElse(this, Clump.value(default))
```

Comment:
I only named this `default` because getOrElse was already used. orElse is consistent with Option.scala